UnitsNet.Tests/CustomCode/RadioactivityTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class RadioactivityTests : RadioactivityTestsBase
+    {
+        // Override properties in base class here
+        protected override double BecquerelsInOneBecquerel => 1;
+
+        protected override double CuriesInOneBecquerel => 2.702702702e-11;
```

Comment:
Verified
https://www.unitconverters.net/radiation-activity/becquerel-to-curie.htm